### ISSUE TITLE
Send me reset password instructions - no email received

### ISSUE THREAD

**Issue by sdalgard** [datagraft/datagraft-portal#82]
No email received when clicking button.
It can be that my local setup doesnt support this (?)

```
Started POST "/users/password" for 10.0.2.2 at 2017-02-13 09:50:53 +0100
Cannot render console from 10.0.2.2! Allowed networks: 127.0.0.0/127.255.255.255, ::1
Processing by Devise::PasswordsController#create as HTML
  Parameters: {"utf8"=>"✓", "user"=>{"email"=>"steffen.sint@gmail.com"}, "commit"=>"Send me reset password instructions"}
  User Load (5.0ms)  SELECT  "users".* FROM "users" WHERE "users"."email" = $1 ORDER BY "users"."id" ASC LIMIT $2  [["email", "steffen.sint@gmail.com"], ["LIMIT", 1]]
  User Load (4.5ms)  SELECT  "users".* FROM "users" WHERE "users"."reset_password_token" = $1 ORDER BY "users"."id" ASC LIMIT $2  [["reset_password_token", "f18e5e59f13ed1e8a5c37d1ad7feaaa164a502a5b2aeed27f816df12be7361c5"], ["LIMIT", 1]]
   (0.5ms)  BEGIN
  SQL (2.7ms)  UPDATE "users" SET "reset_password_token" = $1, "reset_password_sent_at" = $2, "updated_at" = $3 WHERE "users"."id" = $4  [["reset_password_token", "f18e5e59f13ed1e8a5c37d1ad7feaaa164a502a5b2aeed27f816df12be7361c5"], ["reset_password_sent_at", 2017-02-13 08:50:53 UTC], ["updated_at", 2017-02-13 08:50:53 UTC], ["id", 1]]
   (7.0ms)  COMMIT
  Rendering devise/mailer/reset_password_instructions.html.erb
  Rendered devise/mailer/reset_password_instructions.html.erb (1.5ms)
Devise::Mailer#reset_password_instructions: processed outbound mail in 282.0ms
Sent mail to steffen.sint@gmail.com (13.2ms)
Date: Mon, 13 Feb 2017 09:50:53 +0100
To: steffen.sint@gmail.com
Message-ID: <58a1736de706d_c0b3779a40185ce@vb1404-ruby.mail>
Subject: Reset password instructions
Mime-Version: 1.0
Content-Type: text/html;
 charset=UTF-8
Content-Transfer-Encoding: 7bit

<p>Hello steffen.sint@gmail.com!</p>

<p>Someone has requested a link to change your password. You can do this through the link below.</p>

<p><a href="http://localhost:3000/users/password/edit?reset_password_token=5hT1RVFZcHSAi5Y3sL4C">Change my password</a></p>

<p>If you didn't request this, please ignore this email.</p>
<p>Your password won't change until you access the link above and create a new one.</p>

Redirected to http://localhost:3003/users/sign_in
Completed 302 Found in 968ms (ActiveRecord: 19.7ms)


Started GET "/users/sign_in" for 10.0.2.2 at 2017-02-13 09:50:54 +0100
Cannot render console from 10.0.2.2! Allowed networks: 127.0.0.0/127.255.255.255, ::1
Processing by Devise::SessionsController#new as HTML
  Rendering devise/sessions/new.html.slim within layouts/application
  Rendered devise/shared/_links.html.slim (4.9ms)
  Rendered devise/sessions/new.html.slim within layouts/application (18.4ms)
Completed 200 OK in 213ms (Views: 181.8ms | ActiveRecord: 0.0ms)

```

**Comment by nvnikolov** [comment]
You can see if there is an error with the mailer if you set up the environmental variable raise_delivery_errors in the development.rb configuration. Furthermore, it may be misconfigured, so check out http://guides.rubyonrails.org/action_mailer_basics.html#example-action-mailer-configuration.

**Comment by nvnikolov** [comment]
Fixed in c35a621aa272b8b79a7969b9353b6caf56ae776a
